mws/resource_mws_vpc_endpoint_test.go

```diff
@@ -0,0 +1,254 @@
+package mws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databrickslabs/databricks-terraform/common"
+	"github.com/databrickslabs/databricks-terraform/internal/qa"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMwsVPCEndpoint(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test in short mode.")
+	}
+	acctID := qa.GetEnvOrSkipTest(t, "DATABRICKS_ACCOUNT_ID")
+	awsvreID := qa.GetEnvOrSkipTest(t, "AWS_VPC_RELAY_ENDPOINT_ID")
```

Comment:
1. please follow standard test resource naming convention and rename it to `TEST_VPC_RELAY_ENDPOINT_ID`
2. is it `aws_vpc_endpoint`?... could you please add creation of it to https://github.com/databrickslabs/terraform-provider-databricks/blob/master/scripts/modules/aws-mws-common/vpc.tf#L27, https://github.com/databrickslabs/terraform-provider-databricks/blob/master/scripts/mws-integration/main.tf (as `output "test_vpc_relay_endpoint_id"` - all outputs are uppercased and propagated as environment variables to acceptance and integration tests)

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_endpoint